rules/custom-error.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+const findSuperExpression = node => node.type === 'ExpressionStatement' && node.expression.type === 'CallExpression' && node.expression.callee.type === 'Super';
```

Comment:
I hope/think you are confusing the callee type with something else? ASTExplorer shows me `Super`. But might be missing something.

<img width="397" alt="screen shot 2016-11-27 at 13 14 09" src="https://cloud.githubusercontent.com/assets/1913805/20648314/8b3249f4-b4a3-11e6-822e-f3493aabb640.png">